CONTRIBUTING.md

```diff
@@ -100,4 +100,23 @@ Then, re-run
 ```
 pipenv sync --dev
 ```
-to install the relevant dependencies.
\ No newline at end of file
+to install the relevant dependencies.
```

Comment:
This line seems identical to me that the line above. I don't understand why git is not marking them as equal.

Reply:
I think it has to do with the issue that this file was originally missing a "newline" at the end.

Reply:
Talked to Vlad-- seems like he had a similar thing happen to him and he said it was a newline thing, yes.